tests/Functional/Controller/Api/PackageControllerTest.php

```diff
@@ -40,11 +40,12 @@ public function testAuthorizationRequired(): void
             $this->lastResponseBody(),
             '
             {
-                "errors": {
-                    "credentials": [
-                        "Authentication required."
-                    ]
-                }
```

Comment:
:thinking: prev format looks better ... is it forced by something changing?

Reply:
It's a problem to describe schema when having dynamic keys

Reply:
good argument :+1: